ui/src/api/date.js

```diff
@@ -13,44 +14,37 @@ export const fromISO = stringDate => {
 /**
  * Converts luxon Datetime object to ISO date string
  * @param date
- * @returns {DateTime}
+ * @returns {String}
  */
 export const toISO = date => {
-  return date.toISO();
+  if (_hasIn(date, 'toISO')) {
+    return date.toISO();
+  }
+  return date ? date : '';
 };
 
 /**
  *  Serializes date for display in short format
  *  @param {DateTime} date luxon DateTime
- *  @re
+ *  @returns {String}
  */
 export const toShortDateTime = date => {
-  return date.toFormat('yyyy-MM-dd HH:mm');
+  if (_hasIn(date, 'toFormat')) {
+    return date.toFormat('yyyy-MM-dd HH:mm');
+  }
+  return date ? date : '';
 };
 
 /**
  *  Serializes date for display in short format
  *  @param {DateTime} date luxon DateTime
- *  @re
+ *  @returns {String}
  */
 export const toShortDate = date => {
-  return date.toFormat('yyyy-MM-dd');
-};
-
-/**
- *  Serializes date for UTC date in short format used to
- *  normalise dates for REST API
- *  @param {DateTime} date luxon DateTime
- *  @re
- */
-export const toUTCShortDate = date => {
-  if (!(typeof date === 'string' || date instanceof DateTime)) {
-    throw Error('Wrong date type provided');
-  }
-  if (typeof date === 'string') {
-    date = fromISO(date);
+  if (_hasIn(date, 'toFormat')) {
+    return date.toFormat('yyyy-MM-dd');
   }
-  return date.toUTC().toFormat('yyyy-MM-dd');
+  return date ? date : '';
```

Comment:
this might potentially hide problems, when the date is not there when actually it should. how about simply returning  date without changing it?